openapi.yaml

```diff
@@ -118,11 +151,30 @@ components:
         total_fails:
           type: integer
           description: The total number of failures
+    FailedBuilds:
```

Comment:
Can we name this FailedTest or TestFailure?

Reply:
I've renamed FailedBuilds to TestResults and reused TestResult for FailedBuild.